browser-test/utils.js

```diff
@@ -1,34 +1,31 @@
-import {spawn} from "child_process";
+import {Builder} from "selenium-webdriver";
+import {spawnSync} from "child_process";
 import testData from "./testData.json";
 
+const firefox = require("selenium-webdriver/firefox");
+
 const waitTime = 5000;
 const orgSlug = "default";
 
-export const executeCommand = async (command, argv) => {
-  const executor = await spawn(command, [argv]);
-  executor.stdout.on("data", (data) => {
-    process.stdout.write(`${data}\n`);
-  });
-
-  executor.stderr.on("data", (data) => {
-    process.stderr.write(`stderr: ${data}\n`);
-  });
-
-  executor.on("error", (error) => {
-    process.stderr.write(`error: ${error.message}\n`);
-  });
-
-  executor.on("close", (exitCode) => {
-    if (exitCode !== 0) process.exit(exitCode);
-  });
+export const executeCommand = (command, argv) => {
+  const result = spawnSync(command, [argv]);
+
+  if (result.stdout) {
+    process.stdout.write(`${result.stdout}\n`);
+  }
+  if (result.stderr) {
+    process.stderr.write(`${result.stderr}\n`);
+  }
+  if (result.status !== 0) {
+    process.exit(result.status);
+  }
 };
 
-export const getDriver = async (Builder, options) => {
-  const driver = await new Builder()
+export const getDriver = async () => {
+  return new Builder()
     .forBrowser("firefox")
-    .setFirefoxOptions(options)
+    .setFirefoxOptions(new firefox.Options().headless())
     .build();
-  return driver;
```

Comment:
@codesankalp I simplified this. No need to repeat the headless option and `Builder` in each test file.